Backend/app.js

```diff
@@ -13,12 +13,19 @@ import knex from "./config/knex.js";
 
 const app = express();
 app.use(morgan("dev"));
-app.use(cors());
+const corsOptions = {
+  origin: 'http://localhost:3000',
+  credentials: true, // This allows the server to accept credentials like cookies
+};
+app.use(cors(corsOptions));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(cookieParser());
 routes(app,express.Router());
 
+
+
+
 cloudinary.config({
   cloud_name: "dsinpyvxb",
```

Comment:
use environment variables

Reply:
added proper Add .env file for environment variables.you can see the updates in the latest commit.